src/gentropy/assets/schemas/study_locus.json

```diff
@@ -158,7 +158,7 @@
             },
             {
               "metadata": {},
-              "name": "r2Overall",
+              "name": "r2Major",
```

Comment:
I think renaming this column is bad idea. We may return to overall in the future. Moreover changing schema now is very bad idea. I suggest use r2Overall.

Reply:
@DSuveges mentioned this as well, currently reverting those changes